resources/rulesets/eap8/123-javax-to-jakarta-package.windup.yaml

```diff
@@ -3,7 +3,7 @@
   - name: renamed
     nameOfCaptureGroup: renamed
     pattern: javax.(?P<renamed>(activation|annotation|batch|decorator|ejb|el|enterprise|faces|inject|interceptor|jms|json|jws|mail|persistence|resource|security|servlet|transaction|validation|websocket|ws|xml))?.*
-  description: javax.{renamed} has been replaced by jakarta.{renamed}
+  description: The package 'javax' has been replaced by 'jakarta'.
```

Comment:
@pranavgaikwad could you please remove the dot (.) at the end?
It doesn't look right as issue description in UI
![image](https://github.com/konveyor/tackle2-seed/assets/60384172/e19b5e47-6e90-45d0-b25b-a934805f5e8a)